src/__tests__/Snackbar.test.tsx

```diff
@@ -35,7 +40,7 @@ describe('Snackbar', () => {
     const searchInput = screen.getAllByRole('textbox')[0];
     await user.click(searchInput);
 
-    await user.click(screen.getAllByTestId('checkbox-0')[0]);
+    await user.click((await screen.findAllByTestId('checkbox-0'))[0]);
```

Comment:
In most files I changed, I now have to wait for the first change with `findByXXX` functions. I believe this is because the previous mock was returning things synchronously with Promises, while I believe the new mock uses a setTimeout. But this is more correct too.